src/main/resources/application.yaml

```diff
@@ -114,6 +114,9 @@ azure:
     cvp:
       connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
       blob-container-reference: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
+    vh:
```

Comment:
copy paste error? Should they vh_STORAGE_CONTAINER_NAME

Reply:
for now suing same url and containers

Reply:
fixed